### ISSUE TITLE
Working Group meeting on 08/31

### ISSUE THREAD

**Issue by blockstackers** [blockstackers/biz-model_pm#35]
**Date & Time:** Monday, August 31, 2020 @ 2:00pm UTC / 11:00am ET

**Meeting Facilitator:** TBD

**Main Focus:** TBD

**Possible Agenda Topics:**
- #HackStacks: What will the Business Models Working Group be submitting?
- How should we move forward with crowdfunding?: Stackstarter (@MarvinJanssen), app stacking (@njordhov, @russwfranklin)
- Proof of Hodl communities: What are our next steps? What requires further research/exploration? (@pstan26)
- Reading group (open for topic suggestions)

**To Do**

- [x] Establish a meeting facilitator by EOD 08/27
- [ ] Finalize agenda by EOD 08/27
- [ ] Send out event invitations by EOD 08/28

**Comment by friedger** [comment]
Could you rephrase the question?: #HackStacks: What will the Business Models Working Group be submitting?

**Comment by njordhov** [comment]
> app stacking (@njordhov, @russwfranklin)

These are two separate proposals both called _app stacking_. 
The original app stacking proposal is at https://github.com/blockstack/app-mining/issues/229

**Comment by blockstackers** [comment]
@njordhov That's been noted in the Discord discussion. The idea was more that there seems to be a common crowdfunding theme between all of your ideas.

**Comment by blockstackers** [comment]
> Could you rephrase the question?: #HackStacks: What will the Business Models Working Group be submitting?

Sure @friedger. I just wanted to see what some of the Business Model group members are planning to build and submit to #HackStacks. Perhaps, if some people are working on the same idea, they could join forces, etc.
